rest/route/user.go

```diff
@@ -86,6 +86,61 @@ func (h *userSettingsGetHandler) Run(ctx context.Context) gimlet.Responder {
 	return gimlet.NewJSONResponse(apiSettings)
 }
 
+////////////////////////////////////////////////////////////////////////
+//
+// GET /rest/v2/users/{user_id}
+
+type getUserHandler struct {
+	userId string
+}
+
+func makeGetUserHandler() gimlet.RouteHandler {
+	return &getUserHandler{}
+}
+
+// Factory creates an instance of the handler.
+//
+//	@Summary		Get user
```

Comment:
Get user by their user ID or display name.

Does this route need to search by display name? I think any employee is allowed change their display name at any time (and it isn't unique). Using the Evergreen user ID, or their email, seems more stable since those are unique identifiers.

Reply:
Not sure what happened, but it looks like it's using the user ID? I'm fine with that if it works for the papertrail thing.

Reply:
Idk what you mean by "not sure what happened" :eyes: but i'm just going to smack this and see what happens